enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/IndexPersistenceAcceptanceTest.scala

```diff
@@ -165,7 +165,7 @@ class IndexPersistenceAcceptanceTest extends IndexingTestSupport {
 
   test("Should not get new index configuration on database settings changes of WGS84 minimum x extent") {
     // remove the entire western hemisphere
-    val wgs84_x_min = Settings.setting("unsupported.dbms.db.spatial.crs.wgs-84.x.min", Settings.DOUBLE, "0")
+    val wgs84_x_min = SpatialIndexSettings.makeCRSRangeSetting(CoordinateReferenceSystem.WGS84, 0, "min")
```

Comment:
Before you were setting the default value of `wgs-84.x.min = 0`. I don't understand how this is done in the new line. Here `0` seems to imply `x` and the `defaultCartesianExtent` is used?

Reply:
The `"0"` in the previous version was the default value for the setting, something that is not required or useful for this case where we only need the setting as a key for finding and setting a new value (the 0 in the line below). The new version does not set the default at all (removed `"0"`), and the new field `0` is unrelated, meaning only `x` as you already deduced.

Reply:
Ok, that makes sense